docs/docs/interpreter/hdfs.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: page
```

Comment:
could you please bring the format up to date with guideline from [here](https://zeppelin.incubator.apache.org/docs/0.6.0-incubating-SNAPSHOT/development/writingzeppelininterpreter.html), specifically:

Add documentation on how to use your interpreter under docs/interpreter/. Follow the Markdown style as this example. Make sure you list config settings and provide working examples on using your interpreter in code boxes in Markdown. Link to images as appropriate (images should go to docs/assets/themes/zeppelin/img/docs-img/). And add a link to your documentation in the navigation menu (docs/_includes/themes/zeppelin/_navigation.html).